src/plugins/intel_cpu/src/nodes/reorder.h

```diff
@@ -62,18 +62,89 @@ class Reorder : public Node {
     static std::string getReorderArgs(const MemoryDesc &parentDesc, const MemoryDesc &childDesc);
 
     static void reorderData(const IMemory &input, const IMemory &output, MultiCachePtr cache = nullptr);
+    static void reorderData2(const IMemory &input, const IMemory &output, MultiCachePtr cache = nullptr);
```

Comment:
any comments to reorderData2 since the two functions are exactly of the same paramters?

Reply:
Method reorderData2() is old implement and reorderData() is new implement by reorder executor.
We can remove old implement and only keep the new implement.